concrete/single_pages/login.php

```diff
@@ -5,6 +5,7 @@
 defined('C5_EXECUTE') or die('Access denied.');
 
 $form = Loader::helper('form');
+$user = $user ?? app('user');
```

Comment:
Is `'user'` registered?

When executing `app('user')` I have this error:

```
Target class [user] does not exist.
```

Maybe we should use the following instead?

```php
$user = $user ?? app(Concrete\Core\User\User::class);
```